types/reactstrap/lib/Row.d.ts

```diff
@@ -1,8 +1,8 @@
-interface Props {
+interface Props extends React.HTMLProps<any> {
```

Comment:
Hey @pollen8!

If you could specify the typed interface to `React.HTMLProps<HTMLDivElement>` we should merge your PR!